Frontend/updateTranslation.rb

```
@@ -0,0 +1,105 @@
+    end
+
+    puts "Files Downloaded successfully!"
+    file_mapping.each { |key, value| update_strings(key, "#{arg}/src/#{resource}/Localizable/#{value}") }
+    update_constructors("#{arg}/src/#{resource}/Localizable/Types.purs")
+
+  rescue OpenURI::HTTPError => e
+    puts "Error downloading file: #{e.message}"
+  end
+
+  proxy_files.each { |proxy_file| File.delete("#{proxy_file}.txt") }
+end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `main` is too high. [<21, 26, 13> 35.86/23]

(Metrics/AbcSize)